python/cugraph-pyg/cugraph_pyg/tests/int/test_int_cugraph.py

```diff
@@ -0,0 +1,221 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+# Integration test requirements
+"""
+rmm
+cugraph
+cugraph_pyg
+cudf
+dask_cudf
+ogb
+"""
+import rmm
+import cudf
+import dask_cudf
+from cugraph.experimental import PropertyGraph, MGPropertyGraph
+from ogb.nodeproppred import NodePropPredDataset
+
+from cugraph_pyg.data import to_pyg
+from cugraph_pyg.sampler import CuGraphSampler
+
+from torch_geometric.loader import NodeLoader
```

Comment:
Curios if you have added `torch_geometric` under CI or is this not currently run ?

Reply:
This is not currently run.  That will change in the future.